atef/widgets/config/utils.py

```diff
@@ -1362,3 +1379,622 @@ def delete_table_row(self, row: QtWidgets.QWidget) -> None:
                 break
 
         self.table_updated.emit()
+
+
+def set_widget_font_size(widget: QWidget, size: int):
```

Comment:
I'll never stop being surprised that there aren't qt built-ins for stuff like this

Reply:
This is one of the things I copied from your old PR, so you were probably just as surprised back then too 😆